backend/core/Extensions/GeometryExtensions.cs

```diff
@@ -14,7 +14,7 @@ public static class GeometryExtensions
         /// <returns></returns>
         public static Polygon ToPolygon(this Envelope envelope)
         {
-            if (envelope != null && envelope.MinX <= envelope.MaxX && envelope.MinY <= envelope.MaxY)
```

Comment:
what's going on here?

Reply:
After testing I determined it's impossible to have those conditions.  An envelope automatically reorganizes it's X,Y coords so that they don't conflict.  I tried a few dozen attempts to see if I could reach this path, but was unable.